src/modules/communication/grpc_util.cpp

```diff
@@ -56,11 +58,11 @@ bool HeaderController::validateMessage(MessageHeader &msg_header) {
 
 int HeaderController::getJsonSize(MessageHeader &msg_header) {
   int json_size = 0;
```

Comment:
왜 3회 반복하고 쉬프트 연산하고 OR 연산을 하는걸까라는 생각을 하며 코드를 봤어요.
```suggestion
  return ConvertU8ToU32BE(msg_header.total_length);
```
이 작업은 두 개로 나누는게 좋겠어요.

1)  메시지 헤더 중 4B로부터 길이를 구하는 함수 (unit8_t arr -> uint32_t length)
2) getJsonSize 함수

아래 함수를 추가하고요
```
int HeaderController::ConvertU8ToU32BE(uint8_t& len_bytes){
    return static_cast<uint_32t>(len_bytes[3] << 24 | len_bytes[2] << 16 | len_bytes[1] << 8 | len_bytes[0]);
}
```
함수를 두 개로 쪼개면 함수 이름 자체에서 뭘 하는지 말하기 때문에 코드를 이해하는데 1초면 되죠.
헤더가 바이트배열이라서 int로 변환하고 그거를 반환하는구나.
그런데 변환 저렇게 하는거 맞음?